apps/dialtone-documentation/docs/.vuepress/theme/components/Sidebar.vue

```diff
@@ -5,7 +5,7 @@
   <dt-stack
     v-if="sidebarItems[0]?.children.length"
     as="ul"
-    gap="500"
```

Comment:
On the components page, there should be a gap of 500 between the sections.

**Current**
<img width="317" alt="image" src="https://github.com/user-attachments/assets/71a5951f-b776-4188-9b40-bd6d1a69c12c">

**Proposed Update**
<img width="439" alt="image" src="https://github.com/user-attachments/assets/82e7b5ba-8fdb-44b2-9e04-5d4912003516">

Reply:
Fixed